.github/workflows/main.yml

```diff
@@ -24,10 +24,15 @@ jobs:
       - name: Cypress run
         uses: cypress-io/github-action@v2
         with:
+          record: true
```

Comment:
Maybe comment this out and only use it for bugfixing because of the limited free version (and diskspace?)